Azure-Samples/cognitive-services-personalizer-samples
# Add Personalizer Business Demo

## Purpose

* Add [Personalizer Business Demo](https://github.com/southworkscom/cognitive-services-personalizer-samples/blob/dev-next/demos/PersonalizerBusinessDemo/README.md).

![image](https://user-images.githubusercontent.com/42191764/62075379-4d2a7500-b21b-11e9-970f-f1d6adf480e2.png)

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Follow the steps on the [readme](https://github.com/southworkscom/cognitive-services-personalizer-samples/blob/dev-next/demos/PersonalizerBusinessDemo/README.md#to-try-this-sample) or check the latest deploy [here](https://personalizer-devnext.azurewebsites.net/).
